Lib/test/test_abc.py

```diff
@@ -70,6 +70,25 @@ def foo(): return 4
 
 
     class TestABC(unittest.TestCase):
+        def check_isinstance(self, obj, target_class):
+            self.assertIsInstance(obj, target_class)
+            self.assertIsInstance(obj, (target_class,))
+            self.assertIsInstance(obj, target_class | target_class)
```

Comment:
This doesn't test anything useful because `type | type` just returns the type itself. If you want to test the behavior with Union, you'll need to union it with something else.